src/test/skript/tests/regressions/pull-6936-single quotes in commands.sk

```diff
@@ -0,0 +1,7 @@
+command single-quotes-in-commands ["<text="success 2">"]:
+	trigger:
+		broadcast arg
+
+test "single-quotes-in-commands":
+	execute console command "single-quotes-in-commands ""success 1"""
```

Comment:
so the test is considered successful if the broadcast appears?

Reply:
Yes

Reply:
I suppose i could figure out some assertion via the argument? Not sure how that'd work but might be doable

Reply:
you could set a variable in the command and assert that it is set

Reply:
good idea